source/includes/_endpoint-variable.md

```diff
@@ -302,21 +302,23 @@ Behaves sames as PATCH.
 
 `/datasets/{id}/variables/weights/`
 
-GET a "groups" view with a single group named `weight_variables` that contains
-the urls of the variables that have been designated as possible weight variables.
+GET a `shoji:order` that contains the urls of the variables that have been 
+designated as possible weight variables.
 
-PATCH the `value` with a "groups" body. The URLs on the payload (that aren't
-weight variables already) will be appended to the existing list of weight variables.
+PATCH the `graph` with a list of the desired list of weight variables. The
+list will always be overwritten with the new values.
+
+If the dataset has a default weight variable configured, it will always
+be present on the response even if it wasn't included on a PATCH request.
+
+Removing variables from this list will have the side effect of changing any 
+user's preference that had such variables set as their weight to the current
+dataset's default weight.
```

Comment:
I saw a 409 conflict response in the backend-http PR. Please document all possible responses, input validation requirements, etc.